LDKNodeMonday/View/Settings/LSPSettingsView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  LSPSettingsView.swift
```

Comment:
I see file added here, but I think it also needs to be added to project (which would give us a `project.pbxproj` diff in this PR) I think?

I ran into this when I ran your branch locally it was telling me there was no `LSPSettingsView` on line 93 of `SettingsView`. I knew I saw that file in the PR diff so I just had to add the file manually to Xcode, and then had this change locally:

<img width="1453" alt="Screenshot 2025-05-23 at 6 02 21 PM" src="https://github.com/user-attachments/assets/088908d6-3c8a-4771-abbc-ec4b2e6d4102" />